src/atomate2/vasp/schemas/calculation.py

```diff
@@ -672,7 +672,7 @@ def from_vasp_files(
 
         vasprun_kwargs = vasprun_kwargs if vasprun_kwargs else {}
         volumetric_files = [] if volumetric_files is None else volumetric_files
-        vasprun = Vasprun(vasprun_file, **vasprun_kwargs)
+        vasprun = Vasprun(vasprun_file, parse_potcar_file=True, **vasprun_kwargs)
```

Comment:
So what I said about `Structure.charge` vs `Structure._charge` was wrong.
This was the culprit of my HSE caculations running with the wrong charge states.
I think this setting should always be one for vasprun parsing since we are running in a directory that has POTCARs

Reply:
^ This was wrong too it ended up being the CONTCAR copy during the Calculation parsing step.

This is fixed as part of this PR.

https://github.com/materialsproject/atomate2/blob/b3f6f3ef1a3eec261bc3b32bbd1a1263a2d67c22/src/atomate2/vasp/schemas/calculation.py#L500-L504